.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pg_major: ["9.6", "10.9", "11.4", "12.2"]
+        variant: [slim]
+        include:
+          - pg_major: "9.6"
+            postgis_major: "2.4"
+            postgis_version: 2.4.4+dfsg-4.pgdg90+1
+          - pg_major: "10.9"
+            postgis_major: "2.4"
+            postgis_version: 2.4.4+dfsg-4.pgdg90+1
+          - pg_major: "11.4"
+            postgis_major: "2.5"
+            postgis_version: 2.5.4+dfsg-1.pgdg90+1
+          - pg_major: "12.2"
+            postgis_major: "3"
+            postgis_version: 3.0.1+dfsg-4.pgdg100+1
```

Comment:
Needed to update this `postgis_version` since the one in `.travis.yml` was no longer available upstream.